src/map.tsx

```diff
@@ -95,7 +100,17 @@ export class Map extends React.Component<any, any> {
       this.map.on(eventName, olEvents[eventName]);
     }
   }
-
+  // update the view with new props
+  /* Modified by Harinder Randhawa */
+  componentWillReceiveProps(nextProps) {
+    let options = Util.getOptions(Object.assign(this.options, nextProps));
+    if(nextProps.setCenter !== this.props.view.center){
+      this.map.getView().setCenter(nextProps.setCenter);
+    }
+    if(nextProps.setZoom !== this.props.view.zoom){
```

Comment:
This throws errors while running the examples locally. The examples often don't have view defined as a prop. Do we require the view prop or not? If not, put a check in here.